inst/rmarkdown/templates/meningitis_outbreak/skeleton/skeleton.Rmd

```diff
@@ -606,24 +604,27 @@ linelist_cleaned <- linelist_cleaned %>%
 # the tilda (~) is used to assign the new values (Conf, prob, susp, unknown)
 # starting from the specific to the general
 # TRUE assigns all remaining rows 
-# linelist_cleaned <- linelist_cleaned %>%
-#   mutate(case_def = case_when(
-#     test_result == "Positive"                               ~ "Confirmed",
-#     test_result == "Negative" & maculopapular_rash == "Yes" ~ "Probable",
-#     test_result == "Negative"                               ~ "Suspected",
-#     TRUE                                                    ~ "Unknown"
-#   ))
+
+linelist_cleaned <- linelist_cleaned %>%
+  mutate(case_def = case_when(
+    grepl("Mening", pastorex_result)                        ~ "Confirmed",
+    fever == "Yes" &
+      (stiff_neck == "Yes" |
+         petechiae_purpura_at_admission == "Yes")           ~ "Probable",
+    TRUE                                                    ~ "Suspected"
+  ))
 
 
 # Fix logically inconsistent variables
-# example: if no vaccine given previously or if unsure, vaccine doses received
+# example: if no vaccine given previously or if unsure, data of last vax
 # should be NA
-#
-# linelist_cleaned <- linelist_cleaned %>%
-#   mutate(previous_vaccine_doses_received = 
-#            ifelse(grepl("Yes", previously_vaccinated), 
-#                   previous_vaccine_doses_received, NA))
 
+linelist_cleaned <- linelist_cleaned %>%
+  mutate(date_of_last_vaccination =
+           ifelse(grepl("Yes", vaccinated_meningitis_mvc),
```

Comment:
This would be better as a case_when

Reply:
I get this error if I use case_when:
`
Error: must be a `Date` object, not a logical vector
`